### ISSUE TITLE
Manage rust-lang/rust in team repository

### ISSUE THREAD

**Issue by jdno** [rust-lang/simpleinfra#363]
The [rust-lang/team](https://github.com/rust-lang/team) repository provides a sustainable way to manage permissions for repositories on GitHub. The [rust-lang/rust](https://github.com/rust-lang/rust) repository hasn't been added yet, because a large amount of teams and collaborators have access to it. Configuring the repository in the teams repo thus requires careful testing to avoid breaking any integrations or workflows.

Managing permissions by hand is tedious and error-prone, especially when it comes to removing access (which is often forgotten). We therefore want to manage the repository automatically.

This work needs to be done by an administrator who can run the [sync-team](https://github.com/rust-lang/sync-team) locally and iterate on the configuration until it applies without breaking changes.

## Resources

- [Discussion on Zulip about `rust-lang/rust` missing in teams repo](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/Add.20perms.20for.20.60rust-lang.2Frust.60.20for.20T-lang-ops)

**Comment by Kobzol** [comment]
Can you list the `sync-team` missing features here? Or is it really impossible for a non-administrator to help with this? :sweat_smile:

**Comment by Mark-Simulacrum** [comment]
But you can probably do so against a fake org you create, unlike the AWS stuff this is usually easier to spin up.

**Comment by Kobzol** [comment]
Yeah I was thinking that I could just do this against a different GH org.
